UT-Georeferencing-Tin-12d-2

LeeGregory12d opened the pull request:
## Unit test

Tin using IfcMapConversion to translate tin to map coordinates

## Covered topics

*tick one/many*

- [x] project setup
- [x] georeferencing
- [ ] alignment semantics
- [ ] alignment geometry
- [ ] linear placement
- [ ] project breakdown structure / spatial structure
- [ ] terrain
- [x] triangulated irregular network
- [ ] open cross profile
- [ ] sectioned surface & solid
- [ ] pavement & course
- [ ] surface feature
- [ ] earthworks
- [ ] geotechnics
- [ ] longitudinal products (barriers, guardrail, ...)
- [ ] interchange
- [ ] structural
- [ ] drainage
- [ ] signage
- [ ] road furniture
- [ ] rail furniture
- [ ] ports & waterways furniture
- [ ] bridge furniture

LeeGregory12d committed:
UT-Georeferencing-Tin-12d-2

Tin using IfcMapConversion to translate tin to map coordinates

pjanck commented on UT-Georeferencing-Tin-12d-2/readme.md:
```diff
@@ -0,0 +1,36 @@
+
+This scenario covers the additional concepts and/or IFC entities:
+
+- 'IfcMapConversion' is used to translate the data to map coordindates
```
```suggestion
- `IfcMapConversion` is used to translate the data from local engineering to map coordinates
```

LeeGregory12d committed:
Update readme

Replaced apostrophes by grave accents.
Added more information about coordinates

pjanck reviewed (approved):
LFTM.

LeeGregory12d committed:
Update readme - IFC and more clarification on coordinates

pjanck reviewed (approved):
Thank you for the extra clarifications!
